### PR TITLE
chore: add m365 and bot capability telemetry property

### PR DESCRIPTION
**Pull request by XiaofuHuang**
Detail: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/14378265/

1. add `bot-capabilities` and `bot-host-type` to localdebug telemetry
2. add `is-m365` to all cli/vscode telemetry

### CODE REVIEW

**Review by kuojianlu — DISMISSED** [review]
Looks good for the M365 part.

File: packages/vscode-extension/src/handlers.ts [hunk 1]
```diff
@@ -303,6 +304,17 @@ async function getIsFromSample() {
   return undefined;
 }
 
+async function getIsM365() {
+  if (core) {
+    const input = getSystemInputs();
+    input.ignoreEnvInfo = true;
+    await core.getProjectConfig(input);
+
+    return core.isM365;
```
Comment (swatDong) [inline packages/vscode-extension/src/handlers.ts:313]: Is it possible to use projectSettings.isM365? Then no need to update core field.

Comment (XiaofuHuang) [reply packages/vscode-extension/src/handlers.ts]: Removed the changes in core

**Review by swatDong — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]
LGTM

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by kimizhu — APPROVED** [review]

**Pull request merged**